packages/kbn-discover-utils/src/components/custom_control_columns/degraded_docs_control.tsx

```diff
@@ -52,49 +55,63 @@ const degradedDocButtonLabelWhenNotPresent = i18n.translate(
   { defaultMessage: 'All fields in this document were parsed correctly' }
 );
 
-const degradedDocButtonLabelWhenDisabled = i18n.translate(
-  'discover.customControl.degradedDocDisabled',
-  {
-    defaultMessage:
-      'Degraded document field detection is currently disabled for this search. To enable it, include the METADATA directive for the `_ignored` field in your ES|QL query. For example:',
-  }
+const getFormattedCTAMessage = ({ hasClickAction }: { hasClickAction: boolean }) => (
+  <FormattedMessage
+    id="discover.customControl.degradedDocDisabled"
+    defaultMessage="Degraded document field detection is disabled for this search. {action} to add {directive} to your ES|QL query."
+    values={{
+      directive: <EuiCode css={{ display: 'inline-block' }}>METADATA _ignored</EuiCode>,
+      action: hasClickAction
+        ? i18n.translate('discover.customControl.degradedDocDisabled.click', {
+            defaultMessage: 'Click',
+          })
+        : i18n.translate('discover.customControl.degradedDocDisabled.consider', {
+            defaultMessage: 'Consider',
+          }),
+    }}
```

Comment:
It should be two different `discover.customControl.degradedDocDisabled` strings then, otherwise it might be hard to translate `action` inside a sentence to some languages.

Reply:
Addressed with [refactor(discover): change requests](https://github.com/elastic/kibana/pull/195630/commits/5546bf2fdde40582f687cdce9ca4d87b266cb11d)